evalml/tests/pipeline_tests/test_pipelines.py

```diff
@@ -2796,3 +2796,82 @@ def test_training_only_component_in_pipeline_transform(X_y_binary):
     pipeline.fit(X, y)
     transformed = pipeline.transform(X)
     assert len(transformed) == len(X) - 2
+
+
+def test_component_graph_pipeline():
+    component_graph1 = ComponentGraph(
```

Comment:
Just curious, what are each of these three trying to test? Could be useful to name the variables accordingly :)

Reply:
Mm, these are just checking that we can fit the regression, classification, and no estimator component graphs into the pipelines. Can change the name to make it more clear!